eks/kubernetes.tf

```diff
@@ -20,6 +20,22 @@ resource "kubernetes_config_map" "aws_auth" {
   groups:
     - system:bootstrappers
     - system:nodes
+- rolearn: arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/AWSReservedSSO_AD-EKS-Admins_b2abd90bad1696ac
```

Comment:
I don't have the role id, make sure the role matches while deploy `AWSReservedSSO_AD-EKS-Admins_b2abd90bad1696ac`.

Not sure if SSO generate the random id `b2abd90bad1696ac` at the end

Reply:
`arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/AWSReservedSSO_AD-EKS-Admins_b2abd90bad1696ac` should be a variable that is passed. Hence why I said create the following variables in the doc:

- admin_roles (array)
- readonly_roles (array)
- dev_roles (array)
- monitoring_roles (array)

for each role that is passed as

```
admin_roles = [
"arn:aws:iam::1235:role/AWSReservedSSO_AD-EKS-Admins_b2abd90bad1696ac"
]
```

iterate through the role list and generate the output. There may be multiple roles that are admin, etc.